libs/utils/trace_analysis.py

```diff
@@ -950,3 +950,158 @@ def plotCPUFrequencyResidency(self, cpus=None):
                           labels,
                           loc='lower right')
             plt.subplots_adjust(hspace=0.5)
+
+    def getClusterFrequencyResidency(self, cluster):
+        """
+        Get a DataFrame with per cluster frequency residency, i.e. amount of
+        time spent at a given frequency in each cluster.
+
+        :param cluster: list of CPU IDs belonging to the cluster
+        :type cluster: list(int)
+        """
+
+        if not self.trace.hasEvents('cpu_frequency'):
+            logging.warn('Events [cpu_frequency] not found, '\
+                         'plot DISABLED!')
```

Comment:
If we call this method we are just getting two series of active and total time, thus this warning should not refer to a plot.
What about: "cluster active time computation not possible" ?